src/Checker/DeadCode/DeadCodeFinder.php

```diff
@@ -123,6 +123,8 @@ protected function getAllExtendedCoreClasses(string $path): array
             '!*Bootstrap.php',
             '!*Interface.php',
             '!*QueryContainer.php',
+            '!*Repository.php',
+            '!*EntityManager.php',
```

Comment:
Can you please extend the "ValidProject" data set with the Repository and EntityManager files?

Reply:
I see dummy data, but there's no output check